### DISCUSSION TITLE
Are there projects using vavr out there worth take a look at?

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [vavr-io/vavr#2826] [Q&A]
My way of learning is taking a look at others' code and do my best to understand and ask questions. For this I'd like to collect projects using vavr. If you have ideas just drop the links in the comments.

**Comment by EliasDerHai** [comment]
Hi Andras,
Been using vavr for quite a while now at work, which is sth I cannot share ofc. 
Out of boredom I spinned up a [demo project](https://github.com/EliasDerHai/try-out-quarkus) to explore quarkus, jooq and some concepts of DDD over the last 2 weeks. 
It also heavily relies on vavr & vavr-jackson. 
It's a simple demo so don't expect too much, feedback is welcome! 🙃
